ci/pipelines/deploy-to-production.yml

```diff
@@ -0,0 +1,101 @@
+definitions:
+  aws_assumed_role_creds: &aws_assumed_role_creds
+    AWS_ACCESS_KEY_ID: ((.:role.AWS_ACCESS_KEY_ID))
+    AWS_SECRET_ACCESS_KEY: ((.:role.AWS_SECRET_ACCESS_KEY))
+    AWS_SESSION_TOKEN: ((.:role.AWS_SESSION_TOKEN))
+
+  wait_for_deploy_params: &wait_for_deploy_params
+    <<: *aws_assumed_role_creds
+    TAG: ((.:tag))
+    ENVIRONMENT: production-2
+
+  deploy_params: &deploy_params
+    <<: *aws_assumed_role_creds
+    TAG: ((.:tag))
+    ACCOUNT: production
+    ENVIRONMENT: production-2
+
+resources:
+  - name: deploy-to-prod-pipeline-definition
```

Comment:
Not really sure if I should standardise on the naming conventions here, i.e. prod vs production. It might not matter to people either.

Reply:
It's used consistently here for the Concourse names, so fine by me.